### PR TITLE
Added JFR topic

### PR DESCRIPTION
**Pull request by MadelineMurray**
Added JFR topic

### CODE REVIEW

File: docs/HowTo/Troubleshoot/Java-Flight-Recording.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+description: Using Java Flight Recorder with Hyperledger Besu 
+---
+
+# Java Flight Recorder 
+
+[Java Flight Recorder(JFR)](https://docs.oracle.com/javacomponents/jmc-5-4/jfr-runtime-guide/about.htm#JFRUH170)
+is a monitoring tools that collets information about the Java Virtual Machine(JVM) when Hyperledger Besu
```
Comment (benjamincburns) [inline docs/HowTo/Troubleshoot/Java-Flight-Recording.md:8]: small typo: `collets` -> `collects`

File: docs/HowTo/Troubleshoot/Java-Flight-Recording.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+To enable JFR, add the following to the Java runtime arguments: 
+
+`-XX:StartFlightRecording=disk=true,delay=15s,dumponexit=true,filename=/tmp/recording.jfr,maxsize=1024m,maxage=1d,settings=profile,path-to-gc-roots=true`
+
+!!! tip 
+    
+    When recording, cleanly exiting Besu results in better data. If not possible to cleanly exit, 
+    the file may be missing some information not flushed to disk. 
+
+Inspect the file is written to `/tmp/recording.jfr` with tools such as Mission Control.
```
Comment (benjamincburns) [inline docs/HowTo/Troubleshoot/Java-Flight-Recording.md:22]: Might want to link `Mission Control` to Oracle's site for it.

Docs page:
https://docs.oracle.com/javacomponents/jmc-5-5/jmc-user-guide/intro.htm#JMCCI109
Download page:
https://download.oracle.com/technology/products/missioncontrol/updatesites/base/5.2.0/eclipse/

File: docs/HowTo/Troubleshoot/Java-Flight-Recording.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    
+    When recording, cleanly exiting Besu results in better data. If not possible to cleanly exit, 
+    the file may be missing some information not flushed to disk. 
+
+Inspect the file is written to `/tmp/recording.jfr` with tools such as Mission Control. 
+
+!!! important
+    
+    If providing the output file to PegaSys support, some details about the user configuration can 
+    be inferred from the JFR output. Consider any relevant security policies before
```
Comment (benjamincburns) [inline docs/HowTo/Troubleshoot/Java-Flight-Recording.md:27]: Might revise to something like "If providing the output file to PegaSys support, please be aware that while JFR files do not contain secrets such as private keys, some details about the user configuration can be inferred from the JFR output."

File: docs/HowTo/Troubleshoot/Java-Flight-Recording.md (hunk 1 above)
Comment (MadelineMurray) [reply docs/HowTo/Troubleshoot/Java-Flight-Recording.md:8]: Fixed

File: docs/HowTo/Troubleshoot/Java-Flight-Recording.md (hunk 2 above)
Comment (MadelineMurray) [reply docs/HowTo/Troubleshoot/Java-Flight-Recording.md:22]: Added

Comment (macfarla) [reply docs/HowTo/Troubleshoot/Java-Flight-Recording.md:22]: "inspect the file is written" maybe should be "Inspect the file written"

File: docs/HowTo/Troubleshoot/Java-Flight-Recording.md (hunk 3 above)
Comment (MadelineMurray) [reply docs/HowTo/Troubleshoot/Java-Flight-Recording.md]: Fixed

File: docs/HowTo/Troubleshoot/Java-Flight-Recording.md (hunk 2 above)
Comment (MadelineMurray) [reply docs/HowTo/Troubleshoot/Java-Flight-Recording.md]: Fixed

**Pull request merged**